test/Microsoft.Azure.WebJobs.Extensions.Kafka.EndToEndTests/KafkaEndToEndTests.cs

```diff
@@ -509,6 +510,140 @@ public async Task Multiple_Hosts_Process_Events_At_Least_Once()
 
         }
 
+        public List<KafkaEventData<string>> GetKafkaEventsWithTraceParentHeader(int numEvents)
```

Comment:
we can change this to private if the scope is class only

Reply:
Done